cookiecutter.json

```
@@ -5,7 +5,7 @@
   "project_name": "Python Template",
   "project_slug": "{{cookiecutter.project_name.lower().replace(' ', '-').replace('_', '-')}}",
   "package_name": "{{cookiecutter.project_slug.replace('-', '_')}}",
-  "project_short_description": "A cookiecutter template with ARC recommendations.",
+  "project_short_description": "A cookiecutter package with UCL ARC recommendations.",
```

```suggestion
  "project_short_description": "A cookiecutter Python package template with UCL ARC recommendations.",
```
or "A Python package template with UCL ARC recommendations."?

I'm not entirely sure what the intent of `project_short_description` is from the cookiecutter docs, hence my two different suggestions...

---

My rationale for removing `template` from the description is that what is created will not be a template but an instance of the template / a package, and my feeling was having the default short description be `A cookiecutter template with ARC recommendations` might give the impression that running `cookiecutter` will produce a template. For the same reason I initially tried to change the default project name from `Python Template` to `Python package` but that caused tests to fail as we've hardcoded default project name in to the checks in tests and I thought at that point changing this was definitely out of scope for this PR so reverted to original name.

---

👍